src/state/shellscripts/sandbox_exec.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
On the basis that the shell-check/change-all-the-shebangs-style PRs in future might regress this:
```suggestion
#!/bin/bash
# This script is only used on macOS, where /bin/bash is both guaranteed to exist and
# and to be for the native architecture, which is why /usr/bin/env bash is not used.
# See https://github.com/ocaml/opam/issues/5450
```